### ISSUE TITLE
Param pulsar_manager.admin.user/pulsar_manager.admin.password used in the wrong place 

### ISSUE THREAD

**Issue by tabalt** [apache/pulsar-helm-chart#31]
**Describe the bug**
I am try to set pulsar_manager.admin.user/pulsar_manager.admin.password for login pulsar-manager ui, But it's actually used as the account/password for outside PostgreSQL instance.

**To Reproduce**
helm upgrade --install pulsar charts/pulsar --set pulsar_manager.admin.user=pulsarpulsar --set pulsar_manager.admin.password=pulsarpulsar  -n pulsar

```
org.postgresql.util.PSQLException: FATAL: role "pulsarpulsar" does not exist
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2440) ~[postgresql-42.2.5.jar:42.2.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2559) ~[postgresql-42.2.5.jar:42.2.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:133) ~[postgresql-42.2.5.jar:42.2.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:250) ~[postgresql-42.2.5.jar:42.2.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.2.5.jar:42.2.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:195) ~[postgresql-42.2.5.jar:42.2.5]
	at org.postgresql.Driver.makeConnection(Driver.java:454) ~[postgresql-42.2.5.jar:42.2.5]
```

**Comment by sijie** [comment]
@tabalt I think the current pulsar-manager version doesn't support changing the login user and password very well. We are releasing pulsar-manager v0.2.0. It will support setting user and password.

**Comment by vkvm** [comment]
I notice v0.2.0 of pulsar-manager was released, and I tried it a couple of weeks ago. I don't think it supports this yet. The issue appears to be tracked here: https://github.com/apache/pulsar-manager/issues/297 . They do mention a workaround (which I haven't tried).

**Comment by lhotari** [comment]
Please review PR #212, it should fix this issue.

**Comment by lhotari** [comment]
Issue should be resolved by #457
